src/cast.rs

```diff
@@ -266,6 +270,61 @@ impl_to_primitive_uint!(u32);
 impl_to_primitive_uint!(u64);
 impl_to_primitive_uint!(u128);
 
+macro_rules! impl_to_primitive_nonzero_to_method {
+    ($SrcT:ident : $( $(#[$cfg:meta])* fn $method:ident -> $DstT:ident ; )*) => {$(
+        #[inline]
+        $(#[$cfg])*
```

Comment:
I guess you copied the `cfg` from the other macros, but it's vestigial, from the days when `i128` was conditional. We definitely shouldn't propagate that, but could you clean up the others too?

Reply:
I wasn't sure what it was so I just kept it. Removed here and from other macros